### ISSUE TITLE
Can't open HTML files after New Theme command

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#2190]
At some point while working through MainMenubar/basic.html, I lost the ability to open HTML files. I think it happened after I did New Theme..., saw an error dialog, and then reloaded Maqetta. I've tried logging in and out with no success.

```
TypeError
dojo.Deferred.reject.errbackdojo.js:1746
dojo.Deferred.reject.errbackdojo.js:1747
_763.setEditordavinci.js:8615
_172dojo.js:1711
dojo.Deferred._16f.then.thendojo.js:1771
_763.setEditordavinci.js:8609
_c2dojo.js:568
_36dojo.js:646
_16dojo.js:370
_82dojo.js:385
_763.setEditordavinci.js:8570
_3c1._createEditordavinci.js:4885
_3c1.openEditordavinci.js:4845
_7a2._dblClickdavinci.js:8910
_383dojo.js:4555
_492.(anonymous function)dojo.js:6092
_2510._onDblClickdavinci.js:53063
_2510._onDblClickdavinci.js:52744
_383

TypeError
dojo.Deferred.reject.errbackdojo.js:1746
dojo.Deferred.reject.errbackdojo.js:1747
_3c1._createEditordavinci.js:4904
_172dojo.js:1711
dojo.Deferred._16f.then.thendojo.js:1771
_3c1._createEditordavinci.js:4885
_3c1.openEditordavinci.js:4845
_7a2._dblClickdavinci.js:8910
_383dojo.js:4555
_492.(anonymous function)dojo.js:6092
_2510._onDblClickdavinci.js:53063
_2510._onDblClickdavinci.js:52744
_383

Error opening editor for filename: ./project1/file1.html 
TypeError
_3c1.openEditordavinci.js:4853
_172dojo.js:1711
dojo.Deferred._16f.then.thendojo.js:1771
_3c1.openEditordavinci.js:4845
_7a2._dblClickdavinci.js:8910
_383dojo.js:4555
_492.(anonymous function)dojo.js:6092
_2510._onDblClickdavinci.js:53063
_2510._onDblClickdavinci.js:52744
_383dojo.js:4551
```

**Comment by aerwin** [comment]
As a quick follow-up, I created a new project (e.g., "project2') and am able to open HTML files when that project is open. If I switch back to project1, I still cannot open HTML files there.

**Comment by aerwin** [comment]
FYI... confirmed this occurs after trying to create a new theme. E.g., try to clone claro and see alert box with following error when click OK:

```
The page at maqetta.xxx.xxx.com says: undefined.
```

From this point forward, I can't open HTML files.

**Comment by JonFerraiolo** [comment]
-> @billreed63 I'm seeing the same thing with latest source code from github, even when Orion isn't involved.

**Comment by JonFerraiolo** [comment]
@childsb - might be coming your way soon

**Comment by peller** [comment]
my guess is this message is coming from one of the two instances of this line

```
alert(response);
```

in model/resource/Folder.js or Resource.js

The alert's gotta go, as well as the serverJSONRequest call that hides all the relevant error info.

**Comment by billreed63** [comment]
fixed by 944dfe05e6dd177cb96f5959767d7b84660485f5
